.github/workflows/autofix.yml

```diff
@@ -0,0 +1,42 @@
+name: autofix.ci # needed to securely identify the workflow
+
+on:
+  pull_request:
+  push:
+    branches: ["main"]
+permissions:
+  contents: read
+
+jobs:
+  autofix:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: "pnpm"
+
+      - run: pnpm install
+
+      - name: Lint
+        run: pnpm eslint . --fix
```

Comment:
Using eslint to run the lint. We could also use `next lint` which works better for nextjs app but since we are in a monorepo maybe it's better to keep this